src/main/resources/liquibase/workbench_changelog/24_2_0.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+				   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+				   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+		http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+	<changeSet author="cheryleigh" id="v24.2.0-1">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<sqlCheck expectedResult="0">
+					SELECT count(*)
+					FROM permission p, role_type_permission rtp
+					WHERE p.permission_id = rtp.permission_id
+						AND name in ( 'MS_ANALYZE_WITH_STA_BRAPP',
+									'MS_ANALYZE_WITH_DECISION_SUPPORT',
+									'MS_CREATE_GEOREFERENCE',
+									'MS_EDIT_GEOREFERENCE',
+									'GRAPHICAL_QUERIES')
+						AND role_type_id = 3
+				</sqlCheck>
+			</not>
+		</preConditions>
+		<comment>Delete BrAPI permissions for program role type</comment>
+		<sql dbms="mysql" splitStatements="true">
+			DELETE FROM role_type_permission
+			WHERE role_type_id = 3
+			  AND permission_id IN (
+				  SELECT permission_id
+				  FROM permission
+				  WHERE name in ( 'MS_ANALYZE_WITH_STA_BRAPP',
+								  'MS_ANALYZE_WITH_DECISION_SUPPORT',
+								  'MS_CREATE_GEOREFERENCE',
+								  'MS_EDIT_GEOREFERENCE',
+								  'GRAPHICAL_QUERIES'));
+		</sql>
+ 	</changeSet>
+	<changeSet author="cheryleigh" id="v24.2.0-2">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<sqlCheck expectedResult="0">
+					select count(*)
```

Comment:
Minimize the use of subquery:

SELECT COUNT(*)
FROM role_permission AS rp
JOIN role AS r ON rp.role_id = r.id
JOIN permission AS p ON rp.permission_id = p.permission_id
WHERE r.role_type_id = 3
  AND p.name IN ('MS_ANALYZE_WITH_STA_BRAPP',
                 'MS_ANALYZE_WITH_DECISION_SUPPORT',
                 'MS_CREATE_GEOREFERENCE',
                 'MS_EDIT_GEOREFERENCE',
                 'GRAPHICAL_QUERIES')